assets/css/components/_form-authentication.scss

```diff
@@ -0,0 +1,6 @@
+.form-authentication {
+  width: 100%;
+  margin: auto;
+  padding: rem(15px);
+  max-width: rem(400px);
```

Comment:
Minor fix the ordering
```suggestion
  width: 100%;
  max-width: rem(400px);
  padding: rem(15px);
  margin: auto;
```

Reply:
I updated the order of those attributes.